src/Compilers/CSharp/Portable/Symbols/Source/SourceLocalSymbol.cs

```diff
@@ -210,12 +237,13 @@ internal override string GetDebuggerDisplay()
             SyntaxToken identifierToken,
             LocalDeclarationKind declarationKind,
             EqualsValueClauseSyntax initializer = null,
-            Binder initializerBinderOpt = null)
+            Binder initializerBinderOpt = null,
+            bool scopedModifier = false)
```

Comment:
>bool scopedModifier = false [](http://example.com/codeflow?start=12&length=27)

Were all existing call sites examined in case an explicit value should be passed?